scripts/unitTest.sh

```diff
@@ -56,6 +65,7 @@ cd ..
 
 # inject seed data
 cd schulcloud-server
+nvm use
```

Comment:
missing version?

Reply:
should take the version set in .nvmrc file